Issue 17

sergey144010 opened the pull request:
### Is the pull request based off the lastest version?
yes
### What features have you added?
Add type return in getFolders()
### What bugs did you fix?
Issue 17
### Is your code valid PSR-2?
yes
### Has anything in your pull request already been fixed?

### Anything else?

### Optional things below
[] Added your name to the credits
[] Add a update name below:

xXXIMMATTHEWXXx commented:
Looks good to me, if you would can you review my pr then when i merge it will you add the method you added to the methods file?

sergey144010 commented:
I corrected only getFolders() method, and added a new makeArrayFolders() method. I think your and my PR compatible. Must unite without problems. But I already wrote about composer.json and docs/guide. What I think about this -> https://github.com/SSilence/php-imap-client/issues/65

xXXIMMATTHEWXXx commented:
Thanks. I fixed them both in an update for my pr. Merging as i reviewed the changes
